noticeboardapp/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+# Create your models here.
+
+class Notice(models.Model):
+
+    title = models.CharField(max_length=50, default='Notice')
+    user = models.CharField(max_length=50, default='Guest')
+    content = models.CharField(max_length=1000)
```

Comment:
Max lenght is 255 for most databases. If you need more, use TextField. https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.CharField
Actually, postgress can handle unlimited strings, but it's also good practice to use TextField for long strings.

Reply:
@jacoor fixed, thanks :)